tests/terraform/graph/graph_builder/test_graph_builder.py

```diff
@@ -334,6 +335,44 @@ def test_build_graph_for_each_with_variables_and_dynamic_not_crash(self):
         # Shouldn't throw exception
         graph_manager.build_graph_from_source_directory(resources_dir)
 
+    def test_build_rustworkx_graph(self):
```

Comment:
can we create a graph in 2 connectors and see if the numbers are the same ? edges/ nodes

Reply:
we have tests for all flows for the 3 types of graphs